src/export/ExportWavPack.cpp

```diff
@@ -425,6 +486,4 @@ void ExportWavPack::OptionsCreate(ShuttleGui &S, int format)
 
 static Exporter::RegisteredExportPlugin sRegisteredPlugin{ "WavPack",
    []{ return std::make_unique< ExportWavPack >(); }
-};
-
-#endif
+};
```

Comment:
We generally prefer to have an empty new line at the end of the file

Reply:
@crsib I am curious to know why :D